environemnt set up
Running the batch script
2020-12-19 12:17:05.535377: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:17:13.767502: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-12-19 12:17:13.793465: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2900000000 Hz
2020-12-19 12:17:13.797558: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x560b404da820 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-12-19 12:17:13.797600: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-12-19 12:17:13.803301: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2020-12-19 12:17:14.030749: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x560b404dd020 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2020-12-19 12:17:14.030797: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Tesla V100S-PCIE-32GB, Compute Capability 7.0
2020-12-19 12:17:14.036178: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:3b:00.0 name: Tesla V100S-PCIE-32GB computeCapability: 7.0
coreClock: 1.597GHz coreCount: 80 deviceMemorySize: 31.75GiB deviceMemoryBandwidth: 1.03TiB/s
2020-12-19 12:17:14.036252: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:17:14.042747: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:17:14.048038: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-12-19 12:17:14.050160: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-12-19 12:17:14.055495: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-12-19 12:17:14.059316: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-12-19 12:17:14.069195: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-12-19 12:17:14.075671: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-12-19 12:17:14.076976: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:17:15.732400: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-19 12:17:15.732468: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 
2020-12-19 12:17:15.732484: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N 
2020-12-19 12:17:15.738636: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/device:GPU:0 with 30132 MB memory) -> physical GPU (device: 0, name: Tesla V100S-PCIE-32GB, pci bus id: 0000:3b:00.0, compute capability: 7.0)
2020-12-19 12:17:15.744092: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:3b:00.0 name: Tesla V100S-PCIE-32GB computeCapability: 7.0
coreClock: 1.597GHz coreCount: 80 deviceMemorySize: 31.75GiB deviceMemoryBandwidth: 1.03TiB/s
2020-12-19 12:17:15.744155: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:17:15.744222: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:17:15.744248: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-12-19 12:17:15.744268: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-12-19 12:17:15.744288: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-12-19 12:17:15.744307: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-12-19 12:17:15.744327: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-12-19 12:17:15.749604: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-12-19 12:17:15.749642: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-19 12:17:15.749654: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 
2020-12-19 12:17:15.749665: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N 
2020-12-19 12:17:15.754933: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/device:GPU:0 with 30132 MB memory) -> physical GPU (device: 0, name: Tesla V100S-PCIE-32GB, pci bus id: 0000:3b:00.0, compute capability: 7.0)
2020-12-19 12:17:16.228180: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:3b:00.0 name: Tesla V100S-PCIE-32GB computeCapability: 7.0
coreClock: 1.597GHz coreCount: 80 deviceMemorySize: 31.75GiB deviceMemoryBandwidth: 1.03TiB/s
2020-12-19 12:17:16.228237: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:17:16.228275: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:17:16.228296: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-12-19 12:17:16.228312: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-12-19 12:17:16.228327: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-12-19 12:17:16.228345: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-12-19 12:17:16.228369: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-12-19 12:17:16.231070: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-12-19 12:17:16.232989: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:3b:00.0 name: Tesla V100S-PCIE-32GB computeCapability: 7.0
coreClock: 1.597GHz coreCount: 80 deviceMemorySize: 31.75GiB deviceMemoryBandwidth: 1.03TiB/s
2020-12-19 12:17:16.233019: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-12-19 12:17:16.233041: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:17:16.233050: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-12-19 12:17:16.233059: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-12-19 12:17:16.233068: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-12-19 12:17:16.233078: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-12-19 12:17:16.233087: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-12-19 12:17:16.235685: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-12-19 12:17:16.235736: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-19 12:17:16.235743: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 
2020-12-19 12:17:16.235748: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N 
2020-12-19 12:17:16.238828: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 30132 MB memory) -> physical GPU (device: 0, name: Tesla V100S-PCIE-32GB, pci bus id: 0000:3b:00.0, compute capability: 7.0)
Using TensorFlow backend.
WARNING:tensorflow:AutoGraph could not transform <function Model.make_train_function.<locals>.train_function at 0x2b03ff11f488> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: 'arguments' object has no attribute 'posonlyargs'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
Does the system is built with CUDA?: True
Default GPU Device: /device:GPU:0
====> Start running baseline.py
Classes: ['Tomato___Late_blight', 'Tomato___Tomato_Yellow_Leaf_Curl_Virus', 'Peach___healthy', 'Grape___Esca_(Black_Measles)', 'Potato___Late_blight', 'Pepper,_bell___Bacterial_spot', 'Strawberry___healthy', 'Orange___Haunglongbing_(Citrus_greening)', 'Tomato___Leaf_Mold', 'Apple___Black_rot', 'Strawberry___Leaf_scorch', 'Tomato___Early_blight', 'Cherry_(including_sour)___healthy', 'Corn_(maize)___Common_rust_', 'Blueberry___healthy', 'Potato___Early_blight', 'Pepper,_bell___healthy', 'Apple___Cedar_apple_rust', 'Grape___Leaf_blight_(Isariopsis_Leaf_Spot)', 'Tomato___Tomato_mosaic_virus', 'Tomato___Target_Spot', 'Tomato___healthy', 'Peach___Bacterial_spot', 'Corn_(maize)___Cercospora_leaf_spot Gray_leaf_spot', 'Tomato___Spider_mites Two-spotted_spider_mite', 'Corn_(maize)___healthy', 'Squash___Powdery_mildew', 'Cherry_(including_sour)___Powdery_mildew', 'Tomato___Bacterial_spot', 'Grape___Black_rot', 'Apple___healthy', 'Potato___healthy', 'Corn_(maize)___Northern_Leaf_Blight', 'Tomato___Septoria_leaf_spot', 'Raspberry___healthy', 'Soybean___healthy', 'Apple___Apple_scab', 'Grape___healthy']
Number of classes: 38
Total number of images: 54305
first_folder_path: /fs/scratch/PAA0023/dong760/PlantVillage-Dataset/raw/color/Tomato___Late_blight
Number of folder: 38
sample image path: /fs/scratch/PAA0023/dong760/PlantVillage-Dataset/raw/color/Tomato___Late_blight/5fdf3288-d5f4-4c70-9b12-95222b82d415___RS_Late.B 4942.JPG
Image size: (256, 256, 3)
Found 43456 images belonging to 38 classes.
Found 10849 images belonging to 38 classes.
Train size: 43456

====> Statistics: epochs=5, batch_size=256, validation_split=0.2, lr=0.001, momentum=0.9, steps_per_epoch =100, feature shape= (256, 256, 3), no_classes=38, loss_function=categorical_crossentropy
Epoch 1/5
2020-12-19 12:17:32.218676: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-12-19 12:17:37.394779: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
WARNING:tensorflow:AutoGraph could not transform <function Model.make_test_function.<locals>.test_function at 0x2b0c0bf85268> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: 'arguments' object has no attribute 'posonlyargs'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
170/170 - 1164s - loss: 1.0775 - accuracy: 0.7509 - precision: 0.9710 - recall: 0.5457 - auc: 0.9791 - val_loss: 0.4313 - val_accuracy: 0.9008 - val_precision: 0.9676 - val_recall: 0.8030 - val_auc: 0.9982
Epoch 2/5
170/170 - 1088s - loss: 0.3078 - accuracy: 0.9370 - precision: 0.9786 - recall: 0.8663 - auc: 0.9990 - val_loss: 0.2871 - val_accuracy: 0.9317 - val_precision: 0.9694 - val_recall: 0.8781 - val_auc: 0.9991
Epoch 3/5
170/170 - 1079s - loss: 0.1970 - accuracy: 0.9634 - precision: 0.9847 - recall: 0.9244 - auc: 0.9996 - val_loss: 0.2128 - val_accuracy: 0.9499 - val_precision: 0.9731 - val_recall: 0.9116 - val_auc: 0.9995
Epoch 4/5
170/170 - 1066s - loss: 0.1462 - accuracy: 0.9752 - precision: 0.9888 - recall: 0.9499 - auc: 0.9998 - val_loss: 0.1843 - val_accuracy: 0.9560 - val_precision: 0.9781 - val_recall: 0.9286 - val_auc: 0.9996
Epoch 5/5
170/170 - 1080s - loss: 0.1173 - accuracy: 0.9816 - precision: 0.9909 - recall: 0.9636 - auc: 0.9999 - val_loss: 0.1628 - val_accuracy: 0.9590 - val_precision: 0.9783 - val_recall: 0.9371 - val_auc: 0.9996
WARNING:tensorflow:From /users/PAA0023/dong760/miniconda3/envs/tf_latest/lib/python3.6/site-packages/tensorflow/python/training/tracking/tracking.py:111: Model.state_updates (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
2020-12-19 13:49:24.302825: W tensorflow/python/util/util.cc:348] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
WARNING:tensorflow:From /users/PAA0023/dong760/miniconda3/envs/tf_latest/lib/python3.6/site-packages/tensorflow/python/training/tracking/tracking.py:111: Layer.updates (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
WARNING:tensorflow:AutoGraph could not transform <function canonicalize_signatures.<locals>.signature_wrapper at 0x2b0c964a0ae8> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: 'arguments' object has no attribute 'posonlyargs'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
{'loss': [1.077484369277954, 0.3078190088272095, 0.1970018446445465, 0.14619219303131104, 0.11727575212717056], 'accuracy': [0.7509204745292664, 0.9370397925376892, 0.9633652567863464, 0.9751933217048645, 0.9816135764122009], 'precision': [0.9709675908088684, 0.9785541892051697, 0.9847281575202942, 0.9888127446174622, 0.9908894896507263], 'recall': [0.545655369758606, 0.866255521774292, 0.9244062900543213, 0.9498573541641235, 0.9635953903198242], 'auc': [0.9791274666786194, 0.9990442395210266, 0.9996033310890198, 0.9997699856758118, 0.9998564124107361], 'val_loss': [0.4312886595726013, 0.2871438264846802, 0.2127990871667862, 0.18432991206645966, 0.16283145546913147], 'val_accuracy': [0.9008203744888306, 0.9316987991333008, 0.9498571157455444, 0.9560328125953674, 0.9589824080467224], 'val_precision': [0.9675699472427368, 0.9693732261657715, 0.9731378555297852, 0.9780582785606384, 0.9782546162605286], 'val_recall': [0.8030233383178711, 0.8781454563140869, 0.9116047620773315, 0.9285648465156555, 0.9371370673179626], 'val_auc': [0.9981699585914612, 0.999064028263092, 0.9994897246360779, 0.9995588064193726, 0.9996179938316345]}
date and time = 19-12-2020_13-49-16
Saving the model to path: /users/PAA0023/dong760/plant_leaves_diagnosis/saved_models/InceptionV3_BatchSize_256_0.2ValSplit_19-12-2020_13-49-16


p0317.ten.osc.edu:
                                                                               Req'd  Req'd   Elap 
Job id               Username Queue    Name                 SessID NDS   TSK   Memory Time Use S Time 
-------------------- -------- -------- -------------------- ------ ----- ----- ------ ----- - -----
2564371              dong760  gpuseria ondemand/sys/dashboa --         1    28     -- 05:00 R 03:59
2565109              dong760  gpuseria plant_disease_diagno --         1    48     -- 04:00 C 01:32
2565110              dong760  gpuseria plant_disease_diagno --         1    48     -- 04:00 C 01:33
2565113              dong760  gpuseria plant_disease_diagno --         1    48     -- 04:00 C 01:32
2565115              dong760  gpuseria plant_disease_diagno --         1    48     -- 04:00 R 01:32
2565116              dong760  condo-os plant_disease_diagno --         1    48     -- 04:00 R 01:31
2565118              dong760  condo-os plant_disease_diagno --         1    48     -- 04:00 R 01:30
2565119              dong760  condo-os plant_disease_diagno --         1    48     -- 04:00 R 01:30
The date when running current script is :
Sat Dec 19 13:49:45 EST 2020
